### PR TITLE
[Notifier] Add sms77 Notifier Bridge

### PR DESCRIPTION
**Pull request by matthiez**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| License       | MIT
| Doc PR        | [symfony/symfony-docs/pull/15840](https://github.com/symfony/symfony-docs/pull/15840)
| Recipe PR | [symfony/recipes/pull/1003](https://github.com/symfony/recipes/pull/1003)

Add sms77 bridge to Symfony Notifier.
A Germany based SMS provider established in 2003.

### CODE REVIEW

**Review by qdequippe — APPROVED** [review]

File: src/Symfony/Component/Notifier/Bridge/Sms77/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+sms77 Notifier
+=================
+
+Provides [sms77](https://www.sms77.io/) integration for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+SMS//_DSN=sms77://APIKEY@default?from=FROM
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Sms77/README.md]:
```suggestion
SMS77_DSN=sms77://API_KEY@default?from=FROM
```

File: src/Symfony/Component/Notifier/Bridge/Sms77/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+where:
+ - `APIKEY` is your sms77 API key
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Sms77/README.md]:
```suggestion
 - `API_KEY` is your sms77 API key
```

File: src/Symfony/Component/Notifier/Bridge/Sms77/Tests/Sms77TransportFactoryTest.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+        yield [
+            'sms77://host.test?from=TEST',
+            'sms77://apiKey@host.test?from=TEST',
+        ];
+    }
+
+    public function incompleteDsnProvider(): iterable
+    {
+        yield 'missing auth token' => ['sms77://host?from=TEST'];
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Sms77/Tests/Sms77TransportFactoryTest.php]:
```suggestion
        yield 'missing api key' => ['sms77://host?from=TEST'];
```

**Review by OskarStark — CHANGES_REQUESTED** [review]
Only some minor things

File: src/Symfony/Component/Notifier/Bridge/Sms77/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+sms77 Notifier
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Sms77/README.md:1]:
```suggestion
Sms77 Notifier
```

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Please check fabbot and fix the coding standards. Thanks

**Review by javiereguiluz — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @matthiez.

**Pull request closed without merging**